mailbox/store/src/test/java/org/apache/james/mailbox/store/search/AbstractMessageSearchIndexTest.java

```diff
@@ -57,6 +57,8 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AbstractMessageSearchIndexTest.class);
     public static final long LIMIT = 100L;
+    public static final boolean RECENT = true;
```

Comment:
Could be better to introduce a enum to force naming all over the code but maybe that is too much work, isn't it ?